plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -128,21 +142,27 @@ func NewServiceAccount(cl clientset.Interface) *serviceAccount {
 		0,
 	)
 
-	return &serviceAccount{
-		Handler: admission.NewHandler(admission.Create),
-		// TODO: enable this once we've swept secret usage to account for adding secret references to service accounts
-		LimitSecretReferences: false,
-		// Auto mount service account API token secrets
-		MountServiceAccountToken: true,
-		// Reject pod creation until a service account token is available
-		RequireAPIToken: true,
+	a.Run()
```

Comment:
> a bit unexpected to find the Run in a setter. At least I would call SetInternalClientSet something like InitWithClientSet to make clear it is a one-time initialization.

I agree that its weird, but its pre-existing.  I think we want to find a way to plumb a stop channel all the way down and a separate run step, but not part of this.